unipop-core/src/org/unipop/controller/EdgeController.java

```diff
@@ -8,7 +8,14 @@
 import java.util.*;
 
 public interface EdgeController {
-    Iterator<BaseEdge> edges(Predicates predicates, MutableMetrics metrics);
```

Comment:
What was the purpose of it? We couldn't find any usages nor reason for putting it there.